app/views/application/_header_product.html.erb

```diff
@@ -4,7 +4,7 @@
       <%= button_to edit_vendor_product_path(@product.vendor, @product), :method => :get, :class => "btn btn-default" do %>
         <i class="fa fa-fw fa-wrench" aria-hidden="true"></i> Edit Product
       <% end %>
-      <% if mode_atl?%>
+      <% if current_user.user_role?([:atl, :admin]) %>
```

Comment:
Is this meant to not fire in demo mode, then? If the user isn't an ATL or an admin, this will only be true in precert mode, not in demo mode.